standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1454,7 +1454,8 @@ struct GetTablesRequest {
   3: optional ClientCapabilities capabilities,
   4: optional string catName,
   5: optional list<string> processorCapabilities,
-  6: optional string processorIdentifier
+  6: optional string processorIdentifier,
+  7: optional GetProjectionsSpec projectionSpec
```

Comment:
can you add a comment here saying what is support and what is not? For example, ProjectionSpec has fields to filter the parameters too. But we don't support it here. The code should also validate such cases and throw a MetaException with appropriate error message.